pkg/controller/cassandracluster/statefulset.go

```diff
@@ -120,15 +121,37 @@ func statefulSetsAreEqual(sts1, sts2 *appsv1.StatefulSet) bool {
 	//updates to statefulset spec for fields other than 'replicas', 'template', and 'updateStrategy' are forbidden.
 
 	//Things we won't check :
+	//TODO: we should rely on library to ease the comparison between 2 statefulset and to know If we need to apply
```

Comment:
I'm already using another library see https://github.com/Orange-OpenSource/cassandra-k8s-operator/pull/99/files#diff-37aff102a57d3d7b797f152915a6dc16R22

Reply:
I'm keen to open new ticket to deal with this point